examples/kapacitor-ami/config/kapacitor.conf

```diff
@@ -0,0 +1,774 @@
+# The hostname of this node.
+# Must be resolvable by any configured InfluxDB hosts.
+hostname = "localhost"
```

Comment:
Do we run this in Docker? If so, would you need to use 0.0.0.0?

Reply:
This has been parametized in #21